src/core.cpp

```diff
@@ -3,55 +3,112 @@
 #include <random>
 
 #include "core.h"
+#include "action.h"
 
-double Character::GetMoney() {
-    return this->money_;
+RelationshipModifier::RelationshipModifier(Character* target, int start_date, int end_date, double opinion_modifier_) {
+    this->target_ = target;
```

Comment:
More idiomatic would be to use the initalizer list to set these - e.g. `: target_(target), start_date_(start_date), end_date_(end_date), optinion_modifier_(opinion_modifier){}`

Reply:
is that because it'll invoke (conveniently default if no override) copy constructor? so as written it's a little brittle?